File: app/src/main/java/com/aou/ss/MainActivity.java
package com.aou.ss;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import uz.jamshid.lib.WinterLayout;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        WinterLayout winter=(WinterLayout) findViewById(R.id.winter);
        winter.startWinter();
    }

    public void login(View v){
        startActivity(new Intent(this,NewFile.class));
    }


}

File: app/src/main/java/com/aou/ss/MainActivity.kt
package com.aou.ss

import android.content.Intent
import android.os.Bundle
import android.view.View
import com.aou.ss.data.ProjectFile
import com.aou.ss.ui.BaseActivity
import com.aou.ss.ui.UploadActivity
import com.aou.ss.vm.FileAdapter
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : BaseActivity(),MainView {
    lateinit var mainViewModel: MainViewModel
    var allMyFiles=ArrayList<ProjectFile>()
    lateinit var myFilesAdapter: FileAdapter
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        mainViewModel= MainViewModel(this,this)
        events()
    }


    fun events(){
        myFiles.setOnClickListener {
            myFiles.isSelected=true
            sent.isSelected=false
            received.isSelected=false
            mainViewModel.getMyFiles()
        }
        myFiles.callOnClick()
    }

    fun create(v: View?) {
        startActivity(Intent(this, UploadActivity::class.java))
    }

    override fun getMyFilesOnFail(message: String) {
        showMessage(message)
    }

    override fun getMyFilesOnSuccess(files: ArrayList<ProjectFile>) {
        allMyFiles=files
        myFilesAdapter= FileAdapter(this,allMyFiles)
        content.adapter=myFilesAdapter
    }
}